src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -589,10 +583,8 @@ namespace FourPipeBeam {
 
         static constexpr std::string_view routineName("HVACFourPipeBeam::init");
 
-        bool errFlag = false;
```

Comment:
[src/EnergyPlus/HVACFourPipeBeam.cc:592]:(style),[variableScope],The scope of the variable 'errFlag' can be reduced.

Reply:
[src/EnergyPlus/HVACFourPipeBeam.cc:592]:(style),[unreadVariable],Variable 'errFlag' is assigned a value that is never used.